README.md

```diff
@@ -100,335 +143,398 @@ immunopepper make_bg \
+- `--tag-normals`: Default = ''. Name for the normal cohort used for filtering. Needed when there are various normal cohorts. It will be added to the final output name in order to identify against what normal cohort were the samples filtered.
+- `--tag-prefix`: Default = ''. Prefix used for the output files. It is recommended when there are different conditions being studied.
+
+*Optional parameters for normal samples filtering*: Parameters to perform the step 2 of the normal filtering pipeline.
+- `--path-normal-matrix-segm`: Default = None. Path to the matrix of segment expression of kmers in normal samples. This corresponds to output 5 of [output, mode build](#output-files)
+- `--path-normal-matrix-edge`: Default = None. Path to the matrix of junction expression of kmers in normal samples. This corresponds to output 6 of [output, mode build](#output-files)
+- `--n-samples-lim-normal`: Default = None. This is the value for threshold b) in step 2 of normal filtering pipeline. This number will set the number of samples in which we need to see any expression of a specific kmer, i.e. expression > 0, to consider it a normal kmer and exclude it as cancer candidate.
+- `--cohort-expr-support-normal`: Default = None. This is the value for threshold a) in step 2 of normal filtering pipeline. This number corresponds to the normalized expression we need to see in at least one sample in order to consider a kmer as a normal kmer. If a kmer is found with an expression level higher than this threshold in one or more normal samples it will be considered a normal kmer and excluded as cancer candidate.
+
+*Optional parameters for cancer samples filtering*: Parameters to perform the step 2 of the cancer filtering pipeline.
+- `--sample-expr-support-cancer`: #TODO: doesn't have a default. Errors if not provided? This parameter corresponds with the sample specific filtering in the cancer pipeline. The value will correspond to the normalized expression threshold that needs to be reached by each kmer in order to be considered a kmer candidate. Kmers with an expression higher than this threshold in at least one sample will be considered as cancer candidates. #TODO: are samples provided under `--ids_cancer_samples` included in this filtering?
+- `--cohort-expr-support-cancer`: Default = None. This parameter corresponds to the expression threshold in cohort specific filtering. It indicates the normalized expression value that needs to be observed in at least `--n-samples-lim-cancer` in order to consider a kmer as a cancer candidate. Kmers with an expression higher than this threshold in at least `--n-samples-lim-cancer` samples will be considered as cancer candidates. The samples provided under `--ids_cancer_samples` will not be included in cohort filtering.
```

Comment:
This is not exact: The samples provided under `--ids_cancer_samples` will not be included in cohort filtering.

Should be: For each cancer sample of interest provided with `--ids_cancer_samples`, the expression threshold `--cohort-expr-support-cancer` will be assessed in the rest of the cohort, i.e. excluding the sample of interest.

Reply:
Corrected